index.js

```diff
@@ -1,92 +1,82 @@
-// ***** Question 1 *****
-// *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
+// Q1
+let counter = 1
+counter = 2
 // console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
-
-
-// ***** Question 2 *****
-// *** Uncomment the lines below to test
-// *** after testing, comment the line causing the error back in 
-// *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
 
-// console.log(name)
-// => "Raffy"
-// name = "Not Raffy" 
-// => TypeError
-// console.log("%c----------", "color: red") 
 
 
+// Q2
+const name = "Ben"
+//name = "Not Raffy" 
 
-// ***** Question 3 *****
-// *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
 
+// Q3
+function drinkWater(currentThirstLevel) {
+  console.log("Man I sure am thirsty")
+  currentThirstLevel -= 1
+  console.log("Ahh that hits the spot")
+  return currentThirstLevel
+}
+//console.log(drinkWater(12))
 
 
-// ***** Question 4 *****
-// *** Uncomment the lines below to test
-// console.log("%cQuestion 4", "color: red") 
 
+// Q4
+function sameSameButDifferent(num, maybeNum) {
+  if (num === maybeNum)
+    return "same same"
+  else if (num == maybeNum)
+    return "same same (but different)"
+    else return "different"
+}
 // console.log(sameSameButDifferent(5, 5)) 
-// // => "same same"
-
 // console.log(sameSameButDifferent(123, "123")) 
-// // => "same same (but different)"
-
 // console.log(sameSameButDifferent(5, 7)) 
-// // => "different"
-
 // console.log(sameSameButDifferent(123, "122")) 
-// // => "different"
-// console.log("%c----------", "color: red") 
 
 
 
-// ***** Question 5 *****
-// *** Uncomment the lines below to test
-// console.log("%cQuestion 5", "color: red")
+// Q5
+function updateGrade(student, newGrade) {
+  student = student,
+  grade = newGrade
+}
```

Comment:
Student here is an object.  { name: "Duane", grade: 88 }. We want to go into that object and change one of the keys (the key grade) to a different value. We can use dot notation to help us do that.

function updateGrade(student, newGrade) {
  student.grade = newGrade
}